hack/check-pr.sh

```diff
@@ -21,11 +21,20 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+cd `git rev-parse --show-toplevel`
+
 dirs=()
 tests=()
 ref="${1:-HEAD}"
 echo -n "Packages changed since $ref: "
-for d in $(git diff --name-only "$ref" | xargs -n 1 dirname | sort -u); do
+
+changed=$(git diff --name-only "$ref")
+if [[ ${#changed} == 0 ]]; then
+    echo NONE
+    exit 0
```

Comment:
I would argue that we should not exit zero here. We should still run verify stuff (bazel, gofmt, boiler) that do not require a $dirs or $tests flag.

If you do want to exit here then we should print out a message trying to each people how we expect them to use HEAD~N

Reply:
:beers:

Reply:
lol :robot:

Reply:
:beers: